### ISSUE TITLE
'meson build' is failing with error saying it needs a version of wayfire >=0.9.0

### ISSUE THREAD

**Issue by jerblack** [WayfireWM/wayfire-plugins-extra#226]
I am trying to build wayfire-plugins-extra on a Raspberry Pi 5 running 64-bit Raspbian and I'm getting an error on the initial build step.
```
$ sudo apt install libglibmm-2.4-dev libglm-dev libxml2-dev libpango1.0-dev libcairo2-dev wayfire-dev libwlroots-dev libwf-config-dev meson
$ git clone https://github.com/WayfireWM/wayfire-plugins-extra
$ cd wayfire-plugins-extra
$ meson build
meson build
The Meson build system
Version: 1.0.1
Source dir: /home/jeremy/wayfire-plugins-extra
Build dir: /home/jeremy/wayfire-plugins-extra/build
Build type: native build
Project name: wayfire-plugins-extra
Project version: 0.9.0
C compiler for the host machine: cc (gcc 12.2.0 "cc (Debian 12.2.0-14) 12.2.0")
C linker for the host machine: cc ld.bfd 2.40
C++ compiler for the host machine: c++ (gcc 12.2.0 "c++ (Debian 12.2.0-14) 12.2.0")
C++ linker for the host machine: c++ ld.bfd 2.40
Host machine cpu family: aarch64
Host machine cpu: aarch64
Found pkg-config: /usr/bin/pkg-config (1.8.1)
Dependency wayfire found: NO found 0.7.5 but need: '>=0.9.0'
Found CMake: /usr/bin/cmake (3.25.1)
Run-time dependency wayfire found: NO (tried cmake)

meson.build:16:0: ERROR: Dependency lookup for wayfire with method 'pkgconfig' failed: Invalid version, need 'wayfire' ['>=0.9.0'] found '0.7.5'.

A full log can be found at /home/jeremy/wayfire-plugins-extra/build/meson-logs/meson-log.txt
WARNING: Running the setup command as `meson [options]` instead of `meson setup [options]` is ambiguous and deprecated.
```

and here is the final section of the wayfire-plugins-extra/build/meson-logs/meson-log.txt
```
Running compile:
Working directory:  /home/jeremy/wayfire-plugins-extra/build/meson-private/tmpysyrzuwf
Command line:  c++ /home/jeremy/wayfire-plugins-extra/build/meson-private/tmpysyrzuwf/testfile.cpp -o /home/jeremy/wayfire-plugins-extra/build/meson-private/tmpysyrzuwf/output.obj -c -D_FILE_OFFSET_BITS=64 -O0 -fpermissive --print-search-dirs

Code:

Compiler stdout:
 install: /usr/lib/gcc/aarch64-linux-gnu/12/
programs: =/usr/lib/gcc/aarch64-linux-gnu/12/:/usr/lib/gcc/aarch64-linux-gnu/12/:/usr/lib/gcc/aarch64-linux-gnu/:/usr/lib/gcc/aarch64-linux-gnu/12/:/usr/lib/gcc/aarch64-linux-gnu/:/usr/lib/gcc/aarch64-linux-gnu/12/../../../../aarch64-linux-gnu/bin/aarch64-linux-gnu/12/:/usr/lib/gcc/aarch64-linux-gnu/12/../../../../aarch64-linux-gnu/bin/aarch64-linux-gnu/:/usr/lib/gcc/aarch64-linux-gnu/12/../../../../aarch64-linux-gnu/bin/
libraries: =/usr/lib/gcc/aarch64-linux-gnu/12/:/usr/lib/gcc/aarch64-linux-gnu/12/../../../../aarch64-linux-gnu/lib/aarch64-linux-gnu/12/:/usr/lib/gcc/aarch64-linux-gnu/12/../../../../aarch64-linux-gnu/lib/aarch64-linux-gnu/:/usr/lib/gcc/aarch64-linux-gnu/12/../../../../aarch64-linux-gnu/lib/../lib/:/usr/lib/gcc/aarch64-linux-gnu/12/../../../aarch64-linux-gnu/12/:/usr/lib/gcc/aarch64-linux-gnu/12/../../../aarch64-linux-gnu/:/usr/lib/gcc/aarch64-linux-gnu/12/../../../../lib/:/lib/aarch64-linux-gnu/12/:/lib/aarch64-linux-gnu/:/lib/../lib/:/usr/lib/aarch64-linux-gnu/12/:/usr/lib/aarch64-linux-gnu/:/usr/lib/../lib/:/usr/lib/gcc/aarch64-linux-gnu/12/../../../../aarch64-linux-gnu/lib/:/usr/lib/gcc/aarch64-linux-gnu/12/../../../:/lib/:/usr/lib/

Compiler stderr:

Dependency wayfire found: NO found 0.7.5 but need: '>=0.9.0'
Dependency lookup for wayfire with method 'pkgconfig' failed: Invalid version, need 'wayfire' ['>=0.9.0'] found '0.7.5'.
CMake binary for 1 is not cached
CMake binary missing from cross or native file, or env var undefined.
Trying a default CMake fallback at cmake
Found CMake: /usr/bin/cmake (3.25.1)
Extracting basic cmake information
CMake Toolchain: Calling CMake once to generate the compiler state
Calling CMake (['/usr/bin/cmake']) in /home/jeremy/wayfire-plugins-extra/build/meson-private/__CMake_compiler_info__ with:
  - "--trace-expand"
  - "--trace-format=json-v1"
  - "--no-warn-unused-cli"
  - "--trace-redirect=cmake_trace.txt"
  - "-G"
  - "Ninja"
  - "-DCMAKE_TOOLCHAIN_FILE=/home/jeremy/wayfire-plugins-extra/build/meson-private/__CMake_compiler_info__/CMakeMesonTempToolchainFile.cmake"
  - "."
Try CMake generator: auto
Calling CMake (['/usr/bin/cmake']) in /home/jeremy/wayfire-plugins-extra/build/meson-private/cmake_wayfire with:
  - "--trace-expand"
  - "--trace-format=json-v1"
  - "--no-warn-unused-cli"
  - "--trace-redirect=cmake_trace.txt"
  - "-DCMAKE_TOOLCHAIN_FILE=/home/jeremy/wayfire-plugins-extra/build/meson-private/cmake_wayfire/CMakeMesonToolchainFile.cmake"
  - "."
  -- Module search paths:    ['/', '/opt', '/usr', '/usr/local']
  -- CMake root:             /usr/share/cmake-3.25
  -- CMake architectures:    ['aarch64-linux-gnu']
  -- CMake lib search paths: ['lib', 'lib32', 'lib64', 'libx32', 'share', 'lib/aarch64-linux-gnu']
Preliminary CMake check failed. Aborting.
Run-time dependency wayfire found: NO (tried cmake)

meson.build:16:0: ERROR: Dependency lookup for wayfire with method 'pkgconfig' failed: Invalid version, need 'wayfire' ['>=0.9.0'] found '0.7.5'.
```

Any ideas what I can do here? Isn't the latest version of wayfire only 0.8?

**Comment by jerblack** [comment]
Ok, thanks for the heads up. I just figured out that the one module I need (hide_cursor) doesn't seem to be available until 0.8.0, so I need to go see if I can upgrade to that version of Wayfire first. Closing this issue though.

**Comment by eelcohn** [comment]
You could try using this repo until Raspbian has an updated Wayfire version: https://github.com/seffs/wayfire-plugins-extra-raspbian
